### PR TITLE
Fixes for internal RBAC

### PR DESCRIPTION
**Pull request by josejulio**

### CODE REVIEW

File: .rhcicd/clowdapp-backend.yaml [hunk 1]
```diff
@@ -315,6 +317,9 @@ parameters:
 - name: INTERNAL_RBAC_ENABLED
   description: Enable Internal RBAC
   value: "false"
+- name: INTERNAL_RBAC_ADMIN
+  description: Internal admin group
+  value: "crc-notifications-team"
```
Comment (josejulio) [inline .rhcicd/clowdapp-backend.yaml:322]: I'm adding the parameter to be able to test on stage with a non admin user.

File: backend/src/main/java/com/redhat/cloud/notifications/routers/internal/models/AddAccessRequest.java [hunk 2]
```diff
@@ -1,7 +1,11 @@
 package com.redhat.cloud.notifications.routers.internal.models;
 
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
 import java.util.UUID;
 
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
```
Comment (josejulio) [inline backend/src/main/java/com/redhat/cloud/notifications/routers/internal/models/AddAccessRequest.java:8]: `applicationId` as being received as such. Changed to receive it as `application_id`

File: backend/src/main/java/com/redhat/cloud/notifications/models/InternalRoleAccess.java [hunk 3]
```diff
@@ -56,6 +57,7 @@ public void setApplicationId(UUID applicationId) {
         this.applicationId = applicationId;
     }
 
+    @JsonIgnore
     public String getInternalRole() {
         return InternalRoleAccess.getInternalRole(this.role);
```
Comment (josejulio) [inline backend/src/main/java/com/redhat/cloud/notifications/models/InternalRoleAccess.java:62]: this was leaking in the response

File: .rhcicd/clowdapp-backend.yaml (hunk 1 above)
Comment (gwenneg) [reply .rhcicd/clowdapp-backend.yaml:322]: This may be deployed on prod unintentionally, is that OK to commit that value?

Comment (josejulio) [reply .rhcicd/clowdapp-backend.yaml:322]: That's the default admin group. It should match what we have already in the properties file

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**